Index: app/src/main/java/com/example/androidappstorepc/Constant.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/example/androidappstorepc/Constant.java	(date 1608330468996)
+++ app/src/main/java/com/example/androidappstorepc/Constant.java	(date 1608330468996)
@@ -0,0 +1,6 @@
+package com.example.androidappstorepc;
+
+public class Constant {
+    public final static String NAMESHARED="mydata";
+    public final static String ISCONNECTED="isConnected";
+}
